src/azul/plugins/repository/dss/__init__.py

```diff
@@ -103,7 +105,9 @@ def create(cls, catalog: CatalogName) -> RepositoryPlugin:
     @property
     def sources(self) -> AbstractSet[SimpleSourceSpec]:
         assert config.dss_endpoint is not None
-        return {SimpleSourceSpec(name=config.dss_endpoint)}
+        return {
+            SimpleSourceSpec.parse(f'{config.dss_endpoint}:{config.dss_query_prefix}').effective
```

Comment:
@hannes-ucsc 

Is the only reason for using the constructor the line length? If we use the line length, there are not many actual usages of `SimpleSourceSpec.parse`, the only case where its explicitly used is within the canned repository plugin...

Just a heads up, this will also change once we move from `dss_endpoint:dss_query_prefix` to `dss_source`

Reply:
Makes sense.